prow/ANNOUNCEMENTS.md

```diff
@@ -1,6 +1,8 @@
 # Announcements
 
-New features added to each components:
+New features added to each component:
+ - *July 26, 2018* `spyglass` is a pluggable viewing framework for artifacts
+   produced by prowjobs. See a demo [here](prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-large-performance/121)!
```

Comment:
This 404's

Reply:
it 404's because spyglass isnt deployed yet